verilog/analysis/verilog_project.cc

```diff
@@ -240,23 +292,41 @@ const VerilogSourceFile* VerilogProject::LookupFileOrigin(
   return file;
 }
 
-absl::StatusOr<std::vector<std::string>> ParseSourceFileListFromFile(
+FileList ParseSourceFileList(absl::string_view file_list_path,
+                             const std::string& file_list_content) {
+  constexpr absl::string_view kIncludeDirPrefix = "+incdir+";
+  FileList file_list_out;
+  file_list_out.file_list_path = std::string(file_list_path);
+  file_list_out.include_dirs.push_back(".");
+  std::string file_path;
+  std::istringstream stream(file_list_content);
+  while (std::getline(stream, file_path)) {
+    absl::RemoveExtraAsciiWhitespace(&file_path);
+    // Ignore blank lines
+    if (file_path.empty()) continue;
```

Comment:
Are there test-cases for each of these to-be-ignored line types ?

With 
```
MODE=coverage .github/bin/build-and-test.sh
.github/bin/generate-coverage-html.sh
```

... you can run a coverage and look at the resulting HTML (requires `lcov` pacakge to be installed)

(We do run coverage in the CI, but for whatever reason, codecov did not seem to have added it to this PR https://app.codecov.io/gh/chipsalliance/verible )

Reply:
Added a test.